### ISSUE TITLE
Latest Chrome update breaks plugin response handling

### ISSUE THREAD

**Issue by webmasterlv** [jquery-form/form#571]
### Description:
After the latest Chrome update (mine is  83.0.4103.61) the success callback is not called anymore if using `iframe:true` although the request gets to the server. Chrome dev tools shows request as cancelled.
I managed to fix this by setting `iframeSrc:'about:blank'`.
I cannot find a reason or implication of this change. But maybe iframeSrc default should be changed?

### Expected Behavior:
Receive success callback with response from server.

### Actual behavior:
Success callback is not called, request shows aborted in Chrome.

### Versions:
**LoadJSON:**   
**jQuery:**   3.4
**Browsers:**   Chrome  83.0.4103.61

### Demonstration
Demo [here](https://jsfiddle.net/ksbz6n3q/2/)

### Steps to reproduce:
Use default setting of `iframeSrc`

**Comment by phpandrew** [comment]
I am also having the same problem.

To elaborate on @webmasterlv post:
Change:
`options = $.extend(true, {
		url:  url,
		success: $.ajaxSettings.success,
		type: method || 'GET',
		iframeSrc: /^https/i.test(window.location.href || '') ? 'javascript:false' : 'about:blank'
	}, options);`

to
`options = $.extend(true, {
		url:  url,
		success: $.ajaxSettings.success,
		type: method || 'GET',
		iframeSrc: 'about:blank'
	}, options);`

**Comment by webmasterlv** [comment]
I Googled a [little bit](https://stackoverflow.com/questions/4284605/iframe-without-an-src-attribute) about this, and seems like about:blank is the best to be used.
[HTML standard](https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-iframe-element][4.8.5_The_iframe_element]) says:

_If the element has no src attribute specified, or its value is the empty string, let url be the URL "about:blank"._

[There are major updates](https://www.theverge.com/2020/5/19/21262923/google-chrome-privacy-security-safe-browsing-cookies-dns-https-extensions) at the moment, and apparently Chrome does not like javascript:false anymore in the iframe source.

**Comment by longwave** [comment]
Originally this always used `about:blank` but `javascript:false` was added for https URLs in https://github.com/jquery-form/form/commit/ce4324159ecd44cab4779bef6f6d484aef958c5c - but unfortunately the commit doesn't explain why.

**Comment by klonos** [comment]
I'm on 83.0.4103.61 on latest MacOS Catalina and cannot reproduce this. I've used the jsfiddle and both buttons work for me.

**Comment by wordish** [comment]
@klonos both buttons will trigger an alert, but the difference lies in the actual network response. For me, prior to 83, both buttons result in a 404 response from jshell.net. But for 83, the default iframeSrc button request is canceled by the browser, whereas the `about:blank` version sends the request and receives a `404` response.
## default
![Screenshot from 2020-05-25 15-47-20](https://user-images.githubusercontent.com/42463480/82846843-b1e84780-9e9f-11ea-801c-afc37f68b0de.png)

## `about:blank`
![Screenshot from 2020-05-25 15-47-34](https://user-images.githubusercontent.com/42463480/82846846-b6acfb80-9e9f-11ea-8e53-360698f11a59.png)

**Comment by arcataroger** [comment]
(Edit: Oh, nevermind, just saw https://github.com/jquery-form/form/pull/572, which patches this and already accounts for IE)

Just wanted to chime in and say that [over in Drupal land](https://www.drupal.org/project/drupal/issues/3138421), the `about:blank` fix has worked for many of us. We are also unclear about why it was changed to `javascript:false` for https sites, but the workaround seems to pass our tests.

Quick note is that `about:blank` might break this for IE11 in unclear ways. The [Drupal patch](https://www.drupal.org/files/issues/2020-05-24/interdiff-3138421-50-54_d8.txt) added a user agent check to bypass IE.

**Comment by devna13** [comment]
same problem here

using jquery 1.5  and jquery form for document upload
after chrome 83 upgrade 
asp .net forms authentication needs 
```
<authentication mode="Forms">
            <forms requireSSL="true" cookieSameSite="None" .....
```
and document upload needed to be updated to :

 $("#ajaxUploadForm").ajaxForm({
        iframe: true,
        dataType: "json",
        **iframeSrc: "about:blank",
        javascript: false,**
.....

not the best fix but works for now...

**Comment by josepharhar** [comment]
I was linked to this issue from https://bugs.chromium.org/p/chromium/issues/detail?id=1086008
As others have pointed out, the issue is with using javascript: urls, and using about:blank instead is the workaround.
I have a fix in chrome on the way, and it will likely be available in chrome 84.

**Comment by krishnan-unni** [comment]
@josepharhar is this issue same as #573 ?

**Comment by josepharhar** [comment]
> @josepharhar is this issue same as #573 ?

Since that issue says that you must have an iframe with src=“javascript:false”, i think it is the same issue.

**Comment by krishnan-unni** [comment]
@josepharhar Yes we have a iframe created by jquery.form plugin with src as javascript:false. I just can confirmed by modifying the plugin to change the javascript:false to about:blank and it worked.

**Comment by AnanthaRakshith** [comment]
@josepharhar  Please suggest me that I need to do upgrade the jquery plugin with the new version or the chrome developers will going to fix in the next release, if yes please suggest me how I need to handle this case for the time being.

**Comment by josepharhar** [comment]
It will be fixed in chrome within the next three weeks

**Comment by AnanthaRakshith** [comment]
@josepharhar Thank you so much for your update.

**Comment by FortyNinjaFISH** [comment]
Latest Google Chrome update (v83.0.4103.97) appears to have solved this issue for me.

**Comment by josepharhar** [comment]
Yep, I got an update to 83.0.4103.97 on my personal machine today which includes the fix, it is getting rolled out earlier than I originally planned. Check chrome://settings/help to see if you have the update!

**Comment by FortyNinjaFISH** [comment]
Latest Google Chrome update (v83.0.4103.97) appears to have solved this issue for me.

*i removed my previous post as i thought i was on the wrong thread - whoops!*

**Comment by kevindb** [comment]
Fixed by #572
